clustered/client/src/main/java/org/ehcache/clustered/client/internal/store/operations/BaseKeyValueOperation.java

```diff
@@ -25,8 +25,10 @@
 
   private final K key;
   private final V value;
+  private final long timeStamp;
+  private final byte isFirst;
```

Comment:
I would have declared this field as a boolean one and taken care of the value conversion(to 0 or 1) during the encoding/decoding.